lib/prompts/apiMock.ts

```diff
@@ -30,106 +34,65 @@ Your task is to generate a mock API response to the user's request.`,
       role: "user",
       content: `
 I am sending a ${requestMethod} request to the ${path} endpoint${
-        objectNotEmpty({
-          ...queryParameters,
-          ...pathParameters,
-          ...requestBodyParameters,
-        })
+        requestParameters
           ? ` with parameters:
-${JSON.stringify({
-  ...queryParameters,
-  ...pathParameters,
-  ...requestBodyParameters,
-})}`
+${requestParameters.map((param) => chunkToString(param)).join("\n")}`
           : " with no parameters."
       }
 
 ${
-  objectNotEmpty(expectedResponseType)
-    ? `Your response should be JSON of a specific type. Below are 2 examples of how to generate a response from a type.
+  responseType
+    ? `There are specific fields that I want to be returned in the response.
+
+Below is an example of how to generate your response from these fields.
 
 -- EXAMPLE 1 --
```

Comment:
Also, should the examples perhaps have parameters included too? E.g. 2nd one could have `CEO name = Tim Cook` as a parameter?

Reply:
ticketed